include/header.h

```diff
@@ -23,7 +23,7 @@ typedef struct {
 enum ISOMessageReturnValue decodeISOHeader(const char *MessageBuffer, const size_t length,
 											 HeaderType * HeaderData, const char debug);
 
-HeaderType buildISOHeader(enum ISOMessageID id, uint32_t messageLength, const char debug);
+HeaderType buildISOHeader(uint32_t receiverID, uint8_t messageCounter, enum ISOMessageID id, uint32_t messageLength, const char debug);
 
```

Comment:
I would prefer this order of the parameters due to _enum ISOMessageID_ has priority:

HeaderType buildISOHeader(enum ISOMessageID id, uint32_t messageLength, uint32_t receiverID, uint8_t messageCounter, const char debug);